src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -495,9 +436,11 @@
   (let [result (-> {}
                    (maybe-add-http-connector config)
                    (maybe-add-https-connector config)
-                   (assoc :max-threads (determine-max-threads config (num-cpus)))
```

Comment:
We would no longer be auto-adjusting `max-threads` up to the needed server minimum - prior work from TK-130.  The assumption is that the user would now have to set `max-threads` manually in the case that the number of acceptor and selector threads that the server needs to use would exceed the default `max-threads`.  This probably wouldn't be common anyway but is now documented in the `max-threads` section of the Jetty config doc.